src/NServiceBus.Core.Tests/Persistence/InMemory/When_persisting_a_saga_with_complex_types_full_copy_is_created.cs

```diff
@@ -0,0 +1,42 @@
+﻿namespace NServiceBus.Core.Tests.Persistence.InMemory
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+    using Extensibility;
+    using NUnit.Framework;
+
+    [TestFixture]
+    public class When_persisting_a_saga_with_complex_types_full_copy_is_created
```

Comment:
Should this be `When_persisting_a_saga_with_complex_types` instead? Seems like the `full_copy_is_created` could be a test method name instead.